.github/workflows/nemo_tests.yml

```diff
@@ -101,6 +101,18 @@ jobs:
         tail -n 1 output.txt | grep -q "S_max:  0.407622"
         grep -A 1 "Elapsed Time" output.txt
         echo  $GITHUB_REF_NAME $GITHUB_SHA $(grep -A 1 "Elapsed Time" output.txt | head -n 2 | tail -n 1) >> ${NEMO_DIR}/performance_history_openmp_gpu
+        # Upload the results into a Github Gist
+        newresults=$(printf "%q" "$(cat ${NEMO_DIR}/performance_history_openmp_gpu)")
+        {
+          printf '{"description":"New NEMO OpenMP GPU results","files":{"nemo_openmp_gpu.txt":{"content":"'
+          echo -n ${newresults:2:-1}
+          printf '"}}}'
+        }| curl -L -X PATCH \
+             -H "Accept: application/vnd.github+json" \
+             -H "Authorization: Bearer ${{ secrets.GIST_TOKEN }}" \
+             -H "X-GitHub-Api-Version: 2022-11-28" \
+             https://api.github.com/gists/a4049a0fc0a0a11651a5ce6a04d76160 \
```

Comment:
Could we have a link to the gist from a new section in the Wiki?

Reply:
I added a link to the Github Gist in the last step of the reviewer guidelines